packages/renderer/src/lib/dialogs/QuickPickInput.svelte

```diff
@@ -27,6 +27,7 @@ let quickPickSelectedFilteredIndex = 0;
 let quickPickCanPickMany = false;
 
 let inputElement: HTMLInputElement | HTMLTextAreaElement = undefined;
+let comp;
```

Comment:
not sure why we need to use `comp` , we can't use inputElement ?

also probably comp should have the type so we know what it is (also name it probably with 'div' as you're binding a divElement

Reply:
We need something 'up' from the input element, b/c otherwise you could click on places like the title (when there is one) and it would close. This seemed like a reasonable scope to define the bounds of the popup.

Renamed to 'outerDiv' and added type in latest commit.